cookbook/plugins/direct-database-actions/work-with-quantity/readme.md

```diff
@@ -0,0 +1,42 @@
+# plugin-name: "work-with-quantity"
+
+
+# General Introduction
+This plugin creates one flow intended to serve as example for handling Quantity and Unit utilizing SLIMSGate api. 
+
+## Usage: _Content Management_
+## Type: *slimsgate*
+
+# Flow
+
+<details>
+
+<summary>
+
+### <a name="Quantity"></a> Work with Quantity
```

Comment:
Where is this Anchor element intended to take you? Or is it used by another file to link here?